homeassistant/components/homekit/type_media_players.py

```diff
@@ -142,3 +171,184 @@ def update_state(self, new_state):
                               self.entity_id, current_state)
                 self.chars[FEATURE_TOGGLE_MUTE].set_value(current_state)
             self._flag[FEATURE_TOGGLE_MUTE] = False
+
+
+@TYPES.register('TelevisionMediaPlayer')
+class TelevisionMediaPlayer(HomeAccessory):
+    """Generate a Television Media Player accessory."""
+
+    def __init__(self, *args):
+        """Initialize a Switch accessory object."""
+        super().__init__(*args, category=CATEGORY_TELEVISION)
+
+        self._flag = {CHAR_ACTIVE: False, CHAR_ACTIVE_IDENTIFIER: False,
+                      FEATURE_TOGGLE_MUTE: False}
+        self.support_select_source = False
+        self.support_volume_level = False
```

Comment:
Just remove the support flag for: `volume`. I'll leave comments below how to change the current code.